sdk-springboot/src/main/java/io/dapr/springboot/DaprController.java

```diff
@@ -76,7 +77,7 @@ public void healthz() {
    * @param body Raw request body.
    * @return Raw response body.
    */
-  @PutMapping(path = "/actors/{type}/{id}/method/{method}")
+  @PutMapping(path = "/actors/{type}/{id}/method/{method}", produces = MediaType.APPLICATION_JSON_VALUE)

```

Comment:
is this always going to json and not any other type?

Reply:
I double checked and this could be something other than JSON. Let me revert this one.